JitsiConference.js

```diff
@@ -1653,4 +1654,36 @@ JitsiConference.prototype.getSpeakerStats = function() {
     return this.speakerStatsCollector.getStats();
 };
 
+/**
+ * Checks whether video SIP GW service is available.
+ *
+ * @returns {boolean} whether video SIP GW service is available.
+ */
+JitsiConference.prototype.isVideoSIPGWAvailable = function() {
+    return this.videoSIPGWHandler
+        && this.videoSIPGWHandler.isVideoSIPGWAvailable();
+};
+
+/**
+ * Creates a video SIP GW session and returns it if service is enabled.
+ *
+ * @param {string} sipAddress - The sip address to be used.
+ * @param {string} displayName - The display name to be used for this session.
+ * @returns {JitsiVideoSIPGWSession|null}
```

Comment:
It would be good to explain when null can be returned. When looking just at the description I would assume that it's returned when the service is available.

When I actually take a look at the code it looks that VideoSIPGW is created regardless whether it's available or not, but that's less important. I think there should be a private getter with lazy initialisation which should be used from both here and isVideoSIPGWAvailable to create "VideoSIPGW" and be able to check service availability. Currently if 'isVideoSIPGWAvailable' is called, before "createVideoSIPGWSession" it will never return true (no "videoSIPGWHandler" instance).